recipes/tsam/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "tsam" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/FZJ-IEK3-VSA/tsam/archive/49bfc92a2c19c088efd5bdf0fb00e05c7d4383b0.tar.gz
```

Our infrastructure will use the version number to send updates here.

```suggestion
  url: https://github.com/FZJ-IEK3-VSA/tsam/archive/refs/tags/v{{ version }}.tar.gz
```